volatility3/framework/plugins/windows/modules.py

```diff
@@ -127,6 +127,32 @@ def _generator(self):
                 file_output,
             )
 
+    @classmethod
```

Comment:
This is a new exposed method (an additive change to the API), so that means the version should go to `(2, 1, 0)` rather than `(2, 0, 2)`...